src/main/kotlin/com/sipios/springsearch/CriteriaParser.kt

```diff
@@ -24,7 +24,7 @@ class CriteriaParser<T> {
         return visitor.visit(parser.input())
     }
 
-    private fun getParser(queryString: String) : QueryParser {
+    private fun getParser(queryString: String): QueryParser {
```

Comment:
We don't have a linter that detected that?

Reply:
We do now